sections/futures/Trade/DelayedOrderConfirmationModal.tsx

```diff
@@ -291,4 +365,52 @@ const Disclaimer = styled.div`
 	margin-bottom: 12px;
 `;
 
+const OrderSummaryLine = styled.div`
+	display: flex;
+	flex-direction: row;
+	justify-content: space-between;
+	gap: 12px;
+`;
+
+const OrderSummaryCell = styled.div`
```

Comment:
I think we can use `InfoBoxContainer` and `InfoBoxRow` for these instead of creating new styled components.